### ISSUE TITLE
Primary sidebar shrinks when a new window opened

### ISSUE THREAD

**Issue by andreiborisov** [drcika/apc-extension#65]
Every time I reopen a VS Code window, primary sidebar shrinks in width:

Before closing VS Code/its window:
<img width="1248" alt="CleanShot 2023-07-31 at 14 04 35@2x" src="https://github.com/drcika/apc-extension/assets/13354491/74e73758-785e-43ef-87f6-30eadf8204c8">

After closing VS Code/its window:
<img width="1227" alt="CleanShot 2023-07-31 at 14 05 04@2x" src="https://github.com/drcika/apc-extension/assets/13354491/0b295a06-ca5a-409c-8134-615fec834d9d">

It's quite annoying since, after a couple of iterations, it makes the file explorer tiny.

<details>
<summary>Settings</summary>

```jsonc
"apc.activityBar": {
  "size": 77,
  "itemSize": 48,
  "itemMargin": 0
},
"apc.header": {
  "height": 37
},
"apc.electron": {
  "titleBarStyle": "hiddenInset"
},
"apc.sidebar.titlebar": {
  "height": 37
},
"apc.stylesheet": {
  // Make the host button as wide as the .activitybar
  ".statusbar #status\\.host": "width: calc(var(--activity-bar-action-size) - 1px);",
  ".statusbar #status\\.host .codicon": "margin: 0 auto;",
  // Enforce the same color for tabs background
  ".editor-group-container .inline-no-tabs-placeholder": "background-color: rgba(0,0,0,0) !important;",
}
```

</details>

Original discussion: https://github.com/drcika/apc-extension/issues/52#issuecomment-1658070320.

**Comment by drcika** [comment]
it annoys me terribly, 
I keep reloading vscode while working on the extension, 
but unfortunately I haven't found a way to solve that problem, 
cause of the problem is they (vscode team) hardcoded values in different places

**Comment by drcika** [comment]
should be fixed with v0.3.0

**Comment by lehni** [comment]
I can confirm this is fixed by e91ac75b4894801b28678bfc7dbfa8fad920a83c
Great work @drcika !
